workalendar/tests/test_europe.py

```diff
@@ -943,6 +943,18 @@ def test_year_2016(self):
         self.assertIn(date(2016, 12, 6), holidays)
         self.assertIn(date(2016, 12, 8), holidays)
 
+    def test_year_2017(self):
+        holidays = self.cal.holidays_set(2017)
+        self.assertIn(date(2017, 1, 1), holidays)
```

Comment:
Can we turn it into what @ewjohachim suggested in https://github.com/novafloss/workalendar/issues/170#issuecomment-270161567?

```
self.assertEqual(holidays_set(), {
    date(2013, 12, 25),  # Christmas
    # ...
})
```